models/classes/1.3.1.json

```diff
@@ -0,0 +1,164 @@
+{
+  "description": "NcDeviceManager class descriptor",
+  "identity": [
+    1,
+    3,
+    1
+  ],
+  "name": "NcDeviceManager",
+  "fixedRole": "DeviceManager",
+  "properties": [
+    {
+      "description": "Version of nc this dev uses",
```

Comment:
Should "nc" be nca or ncp ?  This dev thinks you should device rather than dev :)

Reply:
Not NCP as that is covered by IS-12 and the version in the urn published in the controls array of a device.
This is possibly the version of MS-05-02 your models are working with. So maybe we should just say that?